### PR TITLE
Dts dell deploy update

### PR DESCRIPTION
**Pull request by TomaszAIR**

### CODE REVIEW

File: docs/variants/dell_optiplex/initial-deployment.md [hunk 1]
```diff
@@ -67,33 +67,22 @@ active. Press F1 to proceed and boot to your Linux system.
...
     ![](../../images/service_mode_warn.jpg)
 
-## Initiali deployment
+## Initial deployment
 
-For simplicity, we are using Dasharo Tools Suite SE version booted from USB.
-Access to DTS SE is available for supporters using [multiple
+For simplicity, we are using Dasharo Tools Suite SE version 1.0.3 booted from
+USB. Access to DTS SE is available for supporters using [multiple
 methods](../../osf-trivia-list/dts/#how-to-get-dasharo-tools-suite-supporters-edition).
```
Comment (sulewskiprzemyslaw) [inline docs/variants/dell_optiplex/initial-deployment.md:74]: Probably, this is not the scope of this task, but this link is empty. Do we accept broken links in published documentation?

File: docs/variants/dell_optiplex/initial-deployment.md [hunk 2]
```diff
@@ -67,33 +67,22 @@ active. Press F1 to proceed and boot to your Linux system.
...
 
-* DTS SE v1.0.0 boots only in UEFI mode
+* After booting to Dasharo Tools Suite SE load Supporter keys (using option `4)`)
+* Next, run `Install Dasharo firmware` by selecting option `2)` and `p)` to
+  accept version
+    - it will prepare a Dasharo HCL report
```
Comment (sulewskiprzemyslaw) [inline docs/variants/dell_optiplex/initial-deployment.md:79]: Do we want to add this information in separate points? It looks a bit weird.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/dell_optiplex/initial-deployment.md (hunk 1 above)
Comment (TomaszAIR) [reply docs/variants/dell_optiplex/initial-deployment.md]: @sulewskiprzemyslaw thank you, fixed https://github.com/Dasharo/docs/pull/352/commits/720abea65ecc9b8ec5b7c4d47315055feae52431#diff-790a9c00e523ff125b88a6094447eb5c3c57a701b736430969c4b6849fa034a1R74

File: docs/variants/dell_optiplex/initial-deployment.md (hunk 2 above)
Comment (TomaszAIR) [reply docs/variants/dell_optiplex/initial-deployment.md]: I added note for that https://github.com/Dasharo/docs/pull/352/commits/720abea65ecc9b8ec5b7c4d47315055feae52431#diff-790a9c00e523ff125b88a6094447eb5c3c57a701b736430969c4b6849fa034a1R80

**Pull request merged**